Shared/Extensions/NSURLExtensions.swift

```diff
@@ -318,13 +318,27 @@ extension URL {
 // Extensions to deal with ReaderMode URLs
 
 extension URL {
+    // Is this an internal ReaderMode url?
```

Comment:
Yup this is the internal version of a reader mode url. Does it make sense to merge in the syncedReadermodeURL check into this?